lib/storage/storage.go

```diff
@@ -195,10 +198,11 @@ func MustOpenStorage(path string, opts OpenOptions) *Storage {
 		retention = retentionMax
 	}
 	s := &Storage{
-		path:           path,
-		cachePath:      filepath.Join(path, cacheDirname),
-		retentionMsecs: retention.Milliseconds(),
-		stopCh:         make(chan struct{}),
+		path:                path,
+		cachePath:           filepath.Join(path, cacheDirname),
+		retentionMsecs:      retention.Milliseconds(),
+		stopCh:              make(chan struct{}),
+		logNewSeriesEndTime: time.Now(),
```

Comment:
Any particular reason to set it to `time.Now` ? I'd leave it the default value, it would ensure the logic is never triggered unless the api call.

Reply:
just make sure logNewSeriesEndTime don't work when process start. I have already make the variable to uint64 and leave it the default value